lib/beaker/host.rb

```diff
@@ -300,7 +300,7 @@ def exec command, options={}
     # @param [String] dir The directory structure to create on the host
     # @return [Boolean] True, if directory construction succeeded, otherwise False
     def mkdir_p dir
-      if host['is_cygwin'].nil? or host['is_cygwin'] == true
+      if @options['is_cygwin'].nil? or @options['is_cygwin'] == true
```

Comment:
Let's switch this to self['is_cygwin'] to make use of the hosts options accessor.